clients/src/main/java/org/apache/kafka/common/security/kerberos/KerberosLogin.java

```diff
@@ -134,127 +134,125 @@ public LoginContext login() throws LoginException {
         // TGT's existing expiry date and the configured minTimeBeforeRelogin. For testing and development,
         // you can decrease the interval of expiration of tickets (for example, to 3 minutes) by running:
         //  "modprinc -maxlife 3mins <principal>" in kadmin.
-        t = KafkaThread.daemon(String.format("kafka-kerberos-refresh-thread-%s", principal), new Runnable() {
```

Comment:
This diff is too hard to interpret. Are these just formatting changes? If there are any logic changes, please call them out.

Reply:
Yes, is this just changing indenting because of the lambda?

Reply:
Yes, just the indentation that is reduced due to lambda.